examples/cpp/ymq/automated_echo_client.cpp

```diff
@@ -7,10 +7,10 @@
 #include <memory>
 #include <string>
 
-#include "scaler/io/ymq/io_context.h"
-#include "scaler/io/ymq/io_socket.h"
-#include "scaler/io/ymq/simple_interface.h"
-#include "scaler/io/ymq/typedefs.h"
+#include "cpp/ymq/io_context.h"
+#include "cpp/ymq/io_socket.h"
+#include "cpp/ymq/simple_interface.h"
+#include "cpp/ymq/typedefs.h"
```

Comment:
Could you make these `#include` omit the `cpp/` directory?

```suggestion
#include "ymq/io_context.h"
#include "mq/io_socket.h"
#include "ymq/simple_interface.h"
#include "ymq/typedefs.h"
```

That way these should match those of the source code.

Reply:
moved to src/cpp/scaler and only using absolute includes now